src/axe-report.ts

```diff
@@ -38,14 +38,33 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
     });
   });
 */
```

Comment:
Here and elsewhere, this comment should stay attached to the function, for the sake of human & machine :)